riscv/tests/riscv_data/keccak/src/lib.rs

```diff
@@ -1,19 +1,32 @@
 #![no_std]
 
-use tiny_keccak::{Hasher, Keccak};
+extern crate alloc;
```

Comment:
I don't think this test should be modified like this, I think we should add a new test instead. It's still good to have a test that uses native tiny keccak.

Reply:
Alrighty. I'm restoring this file entirely and removed all tests using our Rust wrapper because it still depends on `keccak.asm` which is road blocked by Chris' circuit compiler.

I'm moving the tests for Rust wrapper to another draft PR, so that we can merge after all road blocks are cleared.